secure_message/resources/threads.py

```diff
@@ -127,7 +127,24 @@ def _validate_request(request_args, user):
 
 
 class ThreadCounter(Resource):
-    """Get count of all open or closed messages"""
+    """Get count of all conversations for a specific internal user
+    Typically for a specific survey, supports filtering by case, collection exercise, business party id etc
+    Args:
```

Comment:
I know this is picky but throughout all of our other repos we use the sphinx style of docstrings <https://www.datacamp.com/community/tutorials/docstrings-python#fifth-sub> could you convert it to that instead?

Reply:
Disagree . These are not specific parameters so they are not suitable for the normal :param format. What I have done is to move the comments to the get_options function. But even then they are not individual params